virttest/utils_conn.py

```diff
@@ -728,7 +729,12 @@ def conn_recover(self):
                                             server_user, server_pwd,
                                             r"[\#\$]\s*$")
             libvirtd_service = utils_libvirtd.Libvirtd(session=session)
-            libvirtd_service.restart()
+            if libvirt_version.version_compare(5, 6, 0, session):
+                libvirtd_service.stop()
+                session.cmd("systemctl stop libvirtd-tcp.socket")
```

Comment:
Does libvirtd-tcp.socket need start before starting libvirtd ?

Reply:
> Does libvirtd-tcp.socket need start before starting libvirtd ?

It's for recovery, I think libvirtd-tcp.socket should be stopped.

Reply:
But you said in the description that before starting libvirtd, we need start this socket. And now in L735, you did start libvirtd.

Reply:
Okay, that confused you. We have to start socket before libvirtd started to enable tls/tcp. 
but to disable them, no need to start these sockets.

Reply:
Regarding that these sockets are inactive by default, I can accept this change. But could you help update the description to avoid of the confusion?

Reply:
comment is updated.